package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "cloud-search",
+	"version": "1.6.1",
+	"description": "Plugin cloud-search",
+	"main": "index.php",
+	"keywords": [
+		"cloud-search",
+		"javascript",
+		"browserify"
+	],
+	"author": "lando1982",
+	"license": "ISC",
+	"devDependencies": {
+		"grunt": "^1.0.0",
+		"grunt-contrib-copy": "^1.0.0",
+		"grunt-contrib-clean": "^1.0.0",
+		"grunt-contrib-jshint": "^1.0.0",
+		"grunt-contrib-cssmin": "^1.0.1",
+		"grunt-contrib-uglify": "^1.0.1",
+		"grunt-wp-deploy": "^1.3.0",
+		"grunt-wp-readme-to-markdown": "^2.0.1",
+		"matchdep": "*"
```

Comment:
Nothing exciting here just added the additional dependencies needed for the `Gruntfile.js` changes.

Reply:
Oks